lib/money_tracker.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'transactions.rb'
+require 'date'
+
+class MoneyTracker
+
+  attr_reader :balance, :transactions
+
+  def initialize
+    @balance = 0
+    @transactions = Transactions.new
+  end
+
+  def credit(amount)
+    @balance += amount
+    save_transaction(amount, balance)
+  end
+
+  def debit(amount)
```

Comment:
Is it possible for the methods to imitate the domain language in the spec? Generally, if code is "close to the user", it's clearest and most likely to be correct if it mimics the domain language the user uses.